### ISSUE TITLE
onmouseover attribute for achievements with quotation marks in their description is bugged

### ISSUE THREAD

**Issue by GenericMadScientist** [RetroAchievements/RAWeb#889]
An example is the 'Thirsty for Battle' achievement in Pokémon Stadium. The game's page: https://retroachievements.org/game/10181

The element's `onmouseover` attribute has the value
```html
"Tip('<div id=\'objtooltip\' style=\'display:flex;max-width:400px\'><img style=\'margin-right:5px\' src=\'https://s3-eu-west-1.amazonaws.com/i.retroachievements.org/Badge/85910.png\' width=\'64\' height=\'64\' /><div><b>Thirsty for Battle</b><br>Defeat Red in 1P " battle="" now!"&lt;br&gt;&lt;br&gt;3="" points&lt;br&gt;&lt;i&gt;pokemon="" stadium&lt;="" i&gt;&lt;br&gt;&lt;br="" clear="all>Unlocked:" 26="" dec="" 2021,="" 02:21&lt;br&gt;-="HARDCORE=-</div></div>')&quot;"
```

As can be seen, a problem arises once the first quotation mark in the achievement description is encountered. Constrast with the `onmouseover` attribute for the next achievement, Brock:
```html
"Tip('<div id=\'objtooltip\' style=\'display:flex;max-width:400px\'><img style=\'margin-right:5px\' src=\'https://s3-eu-west-1.amazonaws.com/i.retroachievements.org/Badge/85847.png\' width=\'64\' height=\'64\' /><div><b>Brock</b><br>Defeat Pewter Gym Leader Brock<br><br>3 Points<br><i>Pokemon Stadium</i><br><br clear=all>Unlocked: 26 Dec 2021, 03:50<br>-=HARDCORE=-</div></div>')"
```

**Comment by Jamiras** [comment]
This has already been fixed in #882, it just hasn't been deployed yet.

**Comment by luchaos** [comment]
Will be fixed by https://github.com/RetroAchievements/RAWeb/pull/882
